dir.props

```diff
@@ -63,6 +63,8 @@
   <!-- list of nuget package sources passed to nuget.exe -->
   <ItemGroup>
     <NuGetSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-buildtools" />
+    <!-- DO NOT USE THIS SOURCE! It is not retained, and only in this commit for testing. -->
+    <NuGetSourceList Include="https:%2F%2Fdotnet.myget.org/F/nuget-build/api/v3/index.json" />
```

Comment:
Yep, if we go forward with this, I'll copy the NuGet packages to the dotnet-buildtools feed and remove these lines.